assets/content-popup/styles/content-popup.scss

```diff
@@ -198,6 +198,37 @@ $inputs-border-color: #cccccc;
       color: inherit;
       text-decoration: none;
     }
+    .title {
```

Comment:
let's avoid generic naming and redundant nesting, these both are big issues, let's use BEM, it fixes these both